packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/container.tsx

```diff
@@ -73,16 +75,18 @@ const ConfirmResetPasswordContainer = (_: RouteComponentProps) => {
     }
   };
 
-  const checkForRecoveryKey = async () => {
+  const checkForRecoveryKey = async (passwordForgotToken: hexstring) => {
     try {
-      const result: RecoveryKeyCheckResult = await authClient.recoveryKeyExists(
-        undefined,
-        email
-      );
+      const result: RecoveryKeyCheckResult =
+        await authClient.passwordForgotRecoveryKeyStatus(
+          passwordForgotToken,
+          email
```

Comment:
I don't think we need to pass in the email since it's optional and won't be used by the endpoint